docs/synapse/userguides/syn_ref_cmdr.ipynb

```diff
@@ -103,7 +103,7 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "hideCode": false
+    "hideCode": true
```

Comment:
IS there a reason these are being marked to be hidden when they previously were not?